versions/2016-10-31.md

```diff
@@ -409,6 +409,11 @@ Type: S3
 Properties:
   Bucket: my-photo-bucket
   Events: s3:ObjectCreated:*
+  Filter:
+  S3Key:
```

Comment:
Thanks for adding this. However, you need to indent the fields. It should look like this:

```yaml
    Events: s3:ObjectCreated:*
    Filter:
      S3Key:
        Rules:
          - Name: prefix|suffix
            Value: my-prefix|my-suffix
```

Reply:
Sorry wasn't concentrating :/